assets/elm/main.elm

```diff
@@ -306,14 +392,15 @@ view model =
                 -- TODO replace with drawn NameTag
                 , input [ type_ "text", placeholder "enter NameTag", onInput NameTagChange ] []
                 , nameTagView model
-                , button [ onClick (ChangeScreen Game) ] [ text "go to Game" ]
+                , button [ onClick (StartGame (Maybe.withDefault "" model.tableTopic)) ] [ text "go to Game" ]
```

Comment:
just do `onClick StartGame` and move the model topic reading into the update handler

Reply:
Agreed